agimus_controller_ros/agimus_controller_ros/agimus_controller_parameters.yaml

```
@@ -0,0 +1,54 @@
+---
+agimus_controller_params:
+  ocp:
+    dt:
+      type: double
+      default_value: 0.01
+      description: "Step time of the OCP in seconds"
+    horizon_size:
+      type: int
+      default_value: 20
+      description: "Number of nodes of the OCP"
+    armature:
+      type: double_array
+      default_value: [0.1, 0.1, 0.1, 0.1, 0.1, 0.1, 0.1]
+      description: "Armature for each OCP node"
+    gripper_weight:
+      type: double
+      default_value: 500.0
+      description: "Scalar weight for the end-effector pose cost"
+    state_weight:
+      type: double
+      default_value: 5.0
+      description: "Scalar weight for the state cost"
+    control_weight:
+      type: double
+      default_value: 0.001
+      description: "Scalar weight for the control cost"
+    max_qp_iter:
+      type: int
+      default_value: 100
+      description: "Maximum number of qp iterations of the solver"
+    max_iter:
+      type: int
+      default_value: 10
+      description: "Maximum number of iterations of the solver"
+    use_constraints:
+      type: bool
+      default_value: True
+      description: "if True we have Collisions constraints and we use CSQP solver, if False we use FDDP solver"
+    activate_callback:
+      type: bool
+      default_value: False
+      description: "Whether or not we want to use solver callbacks"
+  moving_joint_names:
+    type: string_array
+    description: "The name of the joints we use"
+  publish_debug_datas:
```

```suggestion
  publish_debug_data:
```
`data` is plural already. Singular is `datum`